[BUG] Error shown during runtime: the process cannot access the file because it is being used by another process

**Describe the bug**
On every execution, several errors are displayed in output: _the process cannot access the file because it is being used by another process_
These errors don't affect execution of tests, however they are misleading and attempt against user experience.

**To Reproduce**
Steps to reproduce the behavior:
1. Open shell
2. Execute probr tool `go ./cmd/main.go --varsfile=config.yml`
3. On top of output, see several lines with the following error: `[ERROR] error executing probe: remove probr_output\cucumber\general.json: The process cannot access the file because it is being used by another process`

**Expected behavior**
Only real runtime errors should be displayed in output.
If this is an internal issue with godog framework, it must be handled appropriately.

**Screenshots**
![image](https://user-images.githubusercontent.com/62344673/106616580-b3443680-653b-11eb-9332-43130f4983e6.png)

**Additional context**
Preliminary analysis shows the issue is located in this function: `feature_probr_handler.toFileGodogProbeHandler`

![image](https://user-images.githubusercontent.com/62344673/106618294-492c9100-653d-11eb-8f82-27d1a9e8d6fa.png)

**Aceptance Test**

| Step | Description | Action | Expected Result | Result |
|---|---|---|---|---|
| 1 | Execute binary | run `probr.exe --varsfile=config.yml` | While executing, the following error should not appear in output `[ERROR] error executing probe: remove probr_output\cucumber\<probe>.json: The process cannot access the file because it is being used by another process` | --- |

&#x2705;

--- a/internal/coreengine/feature_probe_handler.go
+++ b/internal/coreengine/feature_probe_handler.go
@@ -28,9 +28,9 @@ func toFileGodogProbeHandler(gd *GodogProbe) (int, *bytes.Buffer, error) {
 	if err != nil {
 		return -1, nil, err
 	}
+
 	status, err := runTestSuite(o, gd)
 
-	//TODO - review!
 	//FUDGE! If the tests are skipped due to tags, then an empty file may
 	//be left lingering.  This will have a non-zero size as we've actually
 	//had to create the file prior to the test run (see line 31).  If it's
@@ -38,7 +38,7 @@ func toFileGodogProbeHandler(gd *GodogProbe) (int, *bytes.Buffer, error) {
 	//and can be removed.
 	i, err := o.Stat()
 	s := i.Size()
-
+	o.Close()
 	if s < 4 {
 		err = os.Remove(o.Name())
 		if err != nil {

--- a/service_packs/kubernetes/pod_security_policy/pod_security_policy.go
+++ b/service_packs/kubernetes/pod_security_policy/pod_security_policy.go
@@ -125,7 +125,7 @@ func (s *scenarioState) runVerificationProbe(c VerificationProbe) error {
 			err = utils.ReformatError("%s: %v - (%v)", utils.CallerName(0), c, res.Err)
 			log.Print(err)
 			return err
-		}
+		} // TODO: Potential bug: (res.Err != nil && res.Internal == false) not handled. E.g: Try to execute 'sudo chroot'.
 
 		//we've managed to execution against the cluster.  This may have failed due to pod security, but this
 		//is still a 'successful' execution.  The exit code of the command needs to be verified against expected

